UPGRADE-5.3.md

```diff
@@ -6,6 +6,11 @@ Asset
 
  * Deprecated `RemoteJsonManifestVersionStrategy`, use `JsonManifestVersionStrategy` instead
 
+DoctrineBridge
+--------------
+
+ * Removed `UuidV*Generator`
```

Comment:
Past tense is fine in the upgrade files?

Reply:
all the file uses this tense

Reply:
Yes I know, I was just asking because Fabien also changed this for the changelog, where all the file also used past tense.
Fine for me 👍

Reply:
Should be changed as well for consistency IMHO

Reply:
Fine to me, updated.